interfaces/apparmor/backend.go

```diff
@@ -829,6 +842,14 @@ func (b *Backend) addContent(securityTag string, snapInfo *snap.Info, cmdName st
 				return `#include if exists "/var/lib/snapd/apparmor/snap-tuning"`
 			}
 			return ""
+		// XXX: Remove this when we have a better solution to including the system
+		// tunables. See snapConfineFromSnapProfile() for a more detailed explanation.
+		case "###INCLUDE_SYSTEM_TUNABLES_HOME_D_WITH_VENDORED_APPARMOR###":
+			features, _ := parserFeatures()
+			if strutil.ListContains(features, "snapd-internal") {
+				return `#include "/etc/apparmor.d/tunables/home.d"`
```

Comment:
same here - this should probably be `#include if exists "/etc/apparmor.d/tunables/home.d"` instead (OR again we can either use the guard `if _, err := os.Stat("/etc/apparmor.d/tunables/home.d/"); err == nil`).

Reply:
same, fixed!